tools/firtool/firtool.cpp

```diff
@@ -102,16 +102,17 @@ processBuffer(std::unique_ptr<llvm::MemoryBuffer> ownedBuffer,
   // If enabled, run the optimizer.
   if (!disableOptimization) {
     // Apply any pass manager command line options.
-    PassManager pm(&context, /*verifyPasses:*/ true);
+    PassManager pm(&context);
     applyPassManagerCLOptions(pm);
 
     pm.addPass(createCSEPass());
     pm.addPass(createCanonicalizerPass());
 
     // Run the lower-to-rtl pass if requested.
     if (lowerToRTL) {
-      OpPassManager &nestedPM = pm.nest<firrtl::CircuitOp>();
-      nestedPM.addPass(firrtl::createLowerFIRRTLToRTLPass());
+      OpPassManager &circuitPM = pm.nest<firrtl::CircuitOp>();
+      OpPassManager &modulePM = circuitPM.nest<firrtl::FModuleOp>();
```

Comment:
After this change https://github.com/llvm/llvm-project/commit/008b9d97cb791b8e3d89c7bc3ddd714ca3a61c80, we need to explicitly nest the pass managers. Without the extra `nest`, we get this nice error message:

```
LLVM ERROR: Can't add pass 'LowerFIRRTLToRTL' restricted to 'firrtl.module' on a PassManager intended to run on 'firrtl.circuit', did you intend to nest?                                                          
```

Reply:
After rebasing, the error is slightly different, but the upshot is the same:
```
LLVM ERROR: Can't add pass 'LowerFIRRTLToRTL' restricted to 'rtl.module' on a PassManager intended to run on 'module', did you intend to nest?                                                                     
```